jax/_src/api.py

```diff
@@ -2328,7 +2328,47 @@ def _device_get(x):
   else:
     return copy()
 
-def device_get(x):
+def device_get(x: Any):
+  """Transfer ``x`` to host.
+
+  Args:
+    x: An array, scalar, DeviceArray or (nested) standard Python container thereof
+      representing the array to be transferred to host.
+
+  Returns:
+    An array or (nested) Python container thereof representing the
+    value of ``x``.
```

Comment:
I believe a line break is required here for the docs to format correctly

Reply:
I am not sure what you mean, the docs looks correct to me. But I added an extra line after line 2340.